docs_user/modules/proc_verifying-the-image-service-adoption.adoc

```diff
@@ -31,15 +31,15 @@ rbd_store_pool=images
 store_description=Ceph glance store backend.
 ----
 
-. If you use a Ceph backend, ensure that the Ceph secrets are properly mounted:
+. If you use a {Ceph} back end, ensure that the {Ceph} secrets are mounted:
 +
 ----
-oc exec -t $GLANCE_POD -c glance-api -- ls /etc/ceph
+$ oc exec -t $GLANCE_POD -c glance-api -- ls /etc/ceph
 ceph.client.openstack.keyring
 ceph.conf
 ----
 
-. Check that the service is active and the endpoints are properly updated in the {OpenStackShort} CLI:
+. Check that the service is active, and that the endpoints are updated in the {OpenStackShort} CLI:
```

Comment:
(Comment from hackfest) Link to comment: https://docs.google.com/document/d/17GzJf8uTfpe7TJrEpf9NomZ98LcnNvUs2V3j9jgDw2w/edit?disco=AAABQ54c_1w

(kgilliga): I think the following link would help: https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0-beta/html-single/deploying_red_hat_openstack_services_on_openshift/index#proc_accessing-the-OpenStackClient-pod_cloud-access-admin

Reply:
Correct, I think it's ok to mention that you can get an `openstack` cli through the `openstackclient` Pod, Can we just refer to that doc to avoid duplicating content?

Reply:
Actually, maybe we could do this instead:

. Check that the service is active, and that the endpoints are updated in the {OpenStackShort} CLI:
+
----
$ oc rsh openstackclient -n openstackclient
$ openstack service list | grep image

| fc52dbffef36434d906eeb99adfc6186 | glance    | image        |

$ openstack endpoint list | grep image

| 569ed81064f84d4a91e0d2d807e4c1f1 | regionOne | glance       | image        | True    | internal  | http://glance-internal-openstack.apps-crc.testing   |
| 5843fae70cba4e73b29d4aff3e8b616c | regionOne | glance       | image        | True    | public    | http://glance-public-openstack.apps-crc.testing     |
| 709859219bc24ab9ac548eab74ad4dd5 | regionOne | glance       | image        | True    | admin     | http://glance-admin-openstack.apps-crc.testing      |
----

Reply:
Works for me (we only need to remove the glance-admin endpoint that is not provided in 18)